benchcab/build_cable.py

```diff
@@ -38,7 +39,7 @@ def add_module_load(lines, nindent, modules):
     return loclines
 
 
-def find_purge_line(filelines, filename=""):
+def find_purge_line(filelines, filename=Path):
```

Comment:
Should the `filename` argument be a positional argument, i.e.
```suggestion
def find_purge_line(filelines, filename: Path):
```
We probably do not need a default value for `filename` here?